features/resources/stateproof/v1_GetProof.json

```diff
@@ -0,0 +1,7 @@
+{
+  "hashtype": "sha512_256",
+  "idx": 0,
+  "proof": "",
+  "stibhash": "1mr5NAjTaheChqu/6dt5JR8yHC8pZW4w8fgOcfSoHEU=",
+  "treedepth": 0
```

Comment:
should we take a non-empty block example?

Reply:
There was a transaction in the block I queried, what else should be in this response?